app/frontend/src/components/AnalysisPanel/AnalysisPanel.tsx

```diff
@@ -29,30 +30,46 @@ export const AnalysisPanel = ({ answer, activeTab, activeCitation, citationHeigh
     const isDisabledCitationTab: boolean = !activeCitation;
     const [citation, setCitation] = useState("");
 
-    const client = useLogin ? useMsal().instance : undefined;
+        const client = useLogin ? useMsal().instance : undefined;
 
-    const fetchCitation = async () => {
-        const token = client ? await getToken(client) : undefined;
-        if (activeCitation) {
-            // Get hash from the URL as it may contain #page=N
-            // which helps browser PDF renderer jump to correct page N
-            const originalHash = activeCitation.indexOf("#") ? activeCitation.split("#")[1] : "";
-            const response = await fetch(activeCitation, {
-                method: "GET",
-                headers: getHeaders(token)
-            });
-            const citationContent = await response.blob();
-            let citationObjectUrl = URL.createObjectURL(citationContent);
-            // Add hash back to the new blob URL
-            if (originalHash) {
-                citationObjectUrl += "#" + originalHash;
+        const fetchCitation = async () => {
```

Comment:
Did all these lines need to change?

Reply:
Ah I see this PR is in draft mode so you may still be working on it.

Reply:
That was a mistake - fixed that, thanks!
The PR is ready to be reviewed, but it's waiting for #1381 so it's still in draft